src/Interpreters/JIT/CHJIT.cpp

```diff
@@ -490,19 +505,52 @@ void CHJIT::runOptimizationPassesOnModule(llvm::Module & module) const
     llvm::CGSCCAnalysisManager cgam;
     llvm::ModuleAnalysisManager mam;
 
+    auto target_analysis = machine->getTargetIRAnalysis();
+    fam.registerPass([&] { return target_analysis; });
+
     llvm::PipelineTuningOptions pto;
     pto.SLPVectorization = true;
+    pto.LoopInterleaving = true;
```

Comment:
I set a breakpoint at l. 512 and ran a JITed query. The debugger showed that the member variables of `pto` already have the same values as in l. 514-520. This means below code has no effect. Did I miss something?

Reply:
You are right, it' OK if not set these values, since already initialized in ctor. I'll change this.